package.yml

```diff
@@ -10,7 +10,7 @@ requires:
     php:
       version: '>=8.1'
     packages:
-       feeds: '>=4.0.0-dev'
```

Comment:
noone knows whether the addon will be compatible with feeds v5 or similar, therefore we should be conservative